content/docs/state-and-lifecycle.md

```diff
@@ -68,23 +68,23 @@ ReactDOM.render(
 );
 ```
 
-To implement this, we need to add "state" to the `Clock` component.
+เพื่อจะพัฒนาในส่วนนี้ เราจำเป็นต้องเพิ่ม "สถานะ"("state") เข้าไปใน `Clock` คอมโพเนนท์
 
-State is similar to props, but it is private and fully controlled by the component.
+สถานะจะมีความคล้ายคลึงกับ props แต่จะมีความเป็นส่วนตัว(private) และสามารถควบคุมได้เต็มที่จากคอมโพเนนท์
 
-## Converting a Function to a Class {#converting-a-function-to-a-class}
+## เปลี่ยนจากฟังค์ชั่น (Function) ให้เป็นคลาส (Class) {#converting-a-function-to-a-class}
```

Comment:
a small suggestion from me.
shall we use ฟัง**ก์**ชัน rather than ฟังค์ชั่น
https://github.com/reactjs/th.reactjs.org/pull/5/files

Reply:
thank you for advise